index.html

```diff
@@ -607,9 +607,19 @@ <h3>Profile Resolution Semantics</h3>
       <h2>Constraints</h2>
       <section id="Document Encoding">
         <h3>Document Encoding</h3>
-        <p>
-          A Document Instance MUST use UTF-8 character encoding as specified in [[UNICODE]].
-        </p>
+        <p>A <a>Document Instance</a> MUST be concretely encoded as a well-formed XML 1.0 [[!xml]] document using the UTF-8
+          character encoding as specified in [[UNICODE]].</p>
+    
+          <p>The resulting [[!xml]] document MUST NOT contain any of the following physical structures:</p>
+    
+          <ul>
+            <li><em>entity declarations</em>; and</li>
+    
+            <li><em>entity references</em> other than to <em>predefined entities</em>.</li>
```

Comment:
Can we add a link to https://www.w3.org/TR/xml/#sec-predefined-ent ? or make it clear that it means only allowing: `&lt;`, `&gt;`, `&amp;`, `&apos;` and `&quot;`?

Reply:
Links added in https://github.com/w3c/dapt/commit/ff6d63ba14522e4273796043a1c2596270ae79d9.

Reply:
I see TTML2 has even a note about that in https://www.w3.org/TR/ttml2/#concrete-encoding with the exact list. Maybe we should have the same?

Reply:
Fine for me, would be helpful probably.

Reply:
Done in 8c2d994.